x/liquidstakeibc/migrations/v2/store.go

```diff
@@ -13,10 +13,20 @@ import (
 // The migration includes:
 //
 // - Migrate host chains to include the Flag attribute.
+// - Migrate host chain validators to include the delegable flag.
+// - Migrate host chain to include the LSM validator cap and the LSM bond factor.
 func MigrateStore(ctx sdk.Context, storeKey storetypes.StoreKey, cdc codec.BinaryCodec) error {
 
 	for _, hc := range getAllHostChains(ctx, storeKey, cdc) {
 		hc.Flags = &types.HostChainFlags{Lsm: false}
+
+		for _, validator := range hc.Validators {
+			validator.Delegable = true
```

Comment:
can you test if this works as expected? 
i believe this one will just update the last element because that's how for loops in go works.. 
better would just be like this 

```suggestion
		for _, validator := range hc.Validators {
			validator.Delegable = true
		for i, _ := range hc.Validators {
			hc.Validators[i] = true
```

Reply:
batch it with the above one to add as single commit